tests/k8st/test_base.py

```diff
@@ -152,6 +152,27 @@ def create_service(self, image, name, ns, port, replicas=1, svc_type="NodePort",
         )
         logger.debug("Service created. status='%s'" % str(api_response.status))
 
+    def create_additional_service(self, name, app, ns, port, svc_type="NodePort", traffic_policy="Local", cluster_ip=None):
```

Comment:
This seems like an exact duplicate to L135-153. Could we refactor `create_service` to also call `create_additional_service` or unless there some divergent logic that I missed?

Reply:
Yep, done!